composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -159,7 +159,7 @@ def add_temporary_platform_extensions(content)
           json = JSON.parse(content)
 
           composer_platform_extensions.each do |extension, requirements|
-            raise "No matching version for #{requirements}!" unless version_for_reqs(requirements)
+            next unless version_for_reqs(requirements)
```

Comment:
Oh yeah no idea why we did this 🤔 is there a test we could add for this new behaviour? 

I think we've been too eager to raise in the past instead of trying to find ways of gracefully degrading/continuing so 👍  on this direction.

Reply:
Yeah I think I can add a meaningful test based on the dry-run I just did, I'd initially opened the PR to check if it wouldn't introduce any failures in CI.